test/utils/kind-k8s-cluster.go

```diff
@@ -112,8 +112,12 @@ func KindNodeImage() string {
 		"1.12": "kindest/node:v1.12.10",
 		"1.13": "kindest/node:v1.13.12",
 		"1.14": "kindest/node:v1.14.10",
-		"1.15": "kindest/node:v1.15.7",
-		"1.16": "kindest/node:v1.16.4",
+		"1.15": "kindest/node:v1.15.12",
+		"1.16": "kindest/node:v1.16.15",
+		"1.17": "kindest/node:v1.17.17",
+		"1.18": "kindest/node:v1.18.15",
+		"1.19": "kindest/node:v1.19.7",
+		"1.20": "kindest/node:v1.20.2",
```

Comment:
I think we should bump "sigs.k8s.io/kind" to 0.10.0 and remove 1.11 - 1.13 here.

Reply:
I've removed obsolete Kubernetes versions, but I can't bump the KIND version since it messes up dependencies.

Reply:
https://github.com/flant/shell-operator/issues/255